### PR TITLE
Fetch data and get list movie

### PR DESCRIPTION
**Pull request by VanHieu7119**
Fetch data and get list movie
Link Redmine: https://edu-redmine.framgia.vn/issues/3716
image

### CODE REVIEW

**Comment by VanHieu7119** [comment]
ready

File: app/src/main/java/com/framgia/moviedb_31/data/source/MovieDataSource.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.framgia.moviedb_31.data.source;
+
+import com.framgia.moviedb_31.data.model.BaseModel;
+import io.reactivex.Flowable;
+
+public interface MovieDataSource {
+    interface MovieRemoteDataSource {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/moviedb_31/data/source/MovieDataSource.java]:
```suggestion
    interface Remote {
```

File: app/src/main/java/com/framgia/moviedb_31/screen/listMovie/ListMovieViewModel.java [hunk 2]
```diff
@@ -1,33 +1,49 @@
...
-            movie.setNameMovie("");
-            movie.setDate("");
-            movie.setRating("");
-            movie.setPoster(String.valueOf(R.drawable.poster2));
-            movie.setContent("");
-            movies.add(movie);
-        }
-        mListMovieAdapter.updateAdapter(movies);
+    private void initData() {
+
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/moviedb_31/screen/listMovie/ListMovieViewModel.java]: remove blank

File: app/src/main/java/com/framgia/moviedb_31/utils/BindingUtils.java [hunk 3]
```diff
@@ -27,7 +27,7 @@ public static void setAdapter(RecyclerView recyclerView, RecyclerView.Adapter ad
     @BindingAdapter("setImagePoster")
     public static void setImagePoster(ImageView view, String url) {
         Glide.with(view.getContext())
-                .load(Integer.parseInt(url))
+                .load(Constant.BASE_IMAGE_URL+url)
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/moviedb_31/utils/BindingUtils.java]: reformat code

**Comment by tuanlvt-1117** [comment]
commented

**Comment by VanHieu7119** [comment]
ready

File: gradle.properties [hunk 4]
```diff
@@ -11,5 +11,4 @@ org.gradle.jvmargs=-Xmx1536m
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
-
-
+API_KEY=819fd4036b78e9083b69a182f338d4d4
```
Comment (tuanlvt-1117) [inline gradle.properties]: hide key

**Comment by tuanlvt-1117** [comment]
commented

**Comment by VanHieu7119** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Comment by tuanlvt-1117** [comment]
merged

**Pull request merged**